plugins/source/azure/resources/services/storage/tables_fetch.go

```diff
@@ -29,3 +29,25 @@ func fetchTables(ctx context.Context, meta schema.ClientMeta, parent *schema.Res
 	}
 	return nil
 }
+
+func getTable(ctx context.Context, meta schema.ClientMeta, res *schema.Resource) error {
+	cl := meta.(*client.Client)
+	svc, err := armstorage.NewTableClient(cl.SubscriptionId, cl.Creds, cl.Options)
+	if err != nil {
+		return err
+	}
+	item := res.Parent.Item.(*armstorage.Account)
+	group, err := client.ParseResourceGroup(*item.ID)
+	if err != nil {
+		return err
+	}
+
+	t := res.Item.(*armstorage.Table)
+
+	v, err := svc.Get(ctx, group, *item.Name, *t.Name, nil)
```

Comment:
`List` only seems to include `properties.tableName` and no `properties.signedIdentifiers`. So we're basically making a second API call just to get those. We could also move it to a different table (`azure_storage_tables_signed_identifiers`?) so that it can be excluded to avoid the extra API call?